README.md

```diff
@@ -1,22 +1,22 @@
 [![cmake workflow](https://github.com/geoscienceaustralia/DynAdjust/actions/workflows/cmake_release.yml/badge.svg)](https://github.com/geoscienceaustralia/DynAdjust/actions/workflows/cmake_release.yml)
 [![cmake tests](https://github.com/geoscienceaustralia/DynAdjust/actions/workflows/test_coverage.yml/badge.svg)](https://github.com/geoscienceaustralia/DynAdjust/actions/workflows/test_coverage.yml)
-[![docker build](https://github.com/icsm-au/DynAdjust/actions/workflows/docker.yml/badge.svg)](https://github.com/icsm-au/DynAdjust/actions/workflows/docker.yml)
-[![codacy badge](https://img.shields.io/codacy/grade/a3944cda0c72445f8a13b1f82b64f714)](https://app.codacy.com/gh/icsm-au/DynAdjust/dashboard)
+[![docker build](https://github.com/GeoscienceAustralia/DynAdjust/actions/workflows/docker.yml/badge.svg)](https://github.com/GeoscienceAustralia/DynAdjust/actions/workflows/docker.yml)
+[![codacy badge](https://img.shields.io/codacy/grade/a3944cda0c72445f8a13b1f82b64f714)](https://app.codacy.com/gh/GeoscienceAustralia/DynAdjust/dashboard)
```

Comment:
Since GA doesn't have an account with codacy yet, this link (https://app.codacy.com/gh/GeoscienceAustralia/DynAdjust/dashboard) will be broken.  That's why this badge still points to icsm (https://app.codacy.com/gh/icsm-au/DynAdjust/dashboard). I suggest leave as icsm-au for now.